io/disk/fiotest.py.data/README

```diff
@@ -0,0 +1,6 @@
+fio is an I/O tool meant to be used both for benchmark and stress/hardware verification.
+
+parameters
+disk: '/dev/sdb' or mpathx or /dev/disk/by-path/dm-uuid-mpathb-xxxxx, /dev/dm-0
+fs: file system type to be created on test disk : ext4
```

Comment:
Shouldn't all File system type being created be mentioned?
ext4, xfs, btrfs, no_fs

Reply:
I will, though yaml has all fs mentioned

Reply:
done